### ISSUE TITLE
Move the hierarchical-link container under the tunnel

### ISSUE THREAD

**Issue by italobusi** [ietf-mpls-yang/te#30]
During the Transport NBI DT discussion, it was not clear why the hierarchical-link container is under the p2p-primary-path container:

```
     |  |  +--rw p2p-primary-paths
     |  |  |  +--rw p2p-primary-path* [name]
     |  |  |     +--rw hierarchical-link
     |  |  |     |  +--rw local-te-node-id?         te-types:te-node-id
     |  |  |     |  +--rw local-te-link-tp-id?      te-types:te-tp-id
     |  |  |     |  +--rw remote-te-node-id?        te-types:te-node-id
     |  |  |     |  +--rw te-topology-identifier
     |  |  |     |     +--rw provider-id?   te-types:te-global-id
     |  |  |     |     +--rw client-id?     te-types:te-global-id
     |  |  |     |     +--rw topology-id?   te-types:te-topology-id

```
It seems that this container should be moved under the tunnel since it should be the tunnel (and not its primary path) to support a link in the client layer topology

See https://github.com/danielkinguk/transport-nbi/pull/29

**Comment by ietf-mpls-yang** [comment]
Need to investigate whether this information should go strictly under tunnel

**Comment by ietf-mpls-yang** [comment]
This should move to under the tunnel
AI (Tarek): close this.

**Comment by ietf-mpls-yang** [comment]
Addressed in 63e261a
